README.md

```diff
@@ -6,9 +6,9 @@ A test-framework-agnostic set of helpers for testing Ember.js applications
 Compatibility
 ------------------------------------------------------------------------------
 
-- Ember 3.8 or above
-- Ember CLI 3.8 or above
-- Node.js 10 or above
+- Ember.js v3.28 or above
+- Ember CLI v3.28 or above
+- Node.js v14 or above
```

Comment:
These changes came from the blueprints. The `peerDependencies` field currently says "ember-source": ">=3.8.0" so I imagine that would need to be updated as well.

Reply:
Update: It turns out ember-source 4.9 drops support for node 12 & 14, so we'll drop support here as well.